stable/msoms/templates/omsagent-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "msoms.fullname" . }}
+  labels:
+    app: {{ template "msoms.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+ updateStrategy:
+    type: RollingUpdate
+ template:
+  metadata:
+   labels:
+    release: {{ .Release.Name }}
+    app: {{ template "msoms.name" .   }}
+   annotations:
+    agentVersion: {{ .Values.omsagent.annotations.agentVersion }}
+    dockerProviderVersion: {{ .Values.omsagent.annotations.dockerProviderVersion }}
+  spec:
+   containers:
+     - name: {{ template "msoms.name" . }}
+       image: "{{ .Values.omsagent.image.registry }}{{ .Values.omsagent.image.org }}/oms:{{ .Values.omsagent.image.tag }}"
```

Comment:
You need to update this since the values in `values.yaml` changed.

Reply:
Missed updating this file.

Reply:
modified the values.yaml file.